bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/common.xml

```diff
@@ -4,6 +4,13 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-group-type id="bridge">
+		<label>API Bridge</label>
+		<channels>
+			<channel id="request-count" typeId="request-count"/>
+		</channels>
+	</channel-group-type>
```

Comment:
I don't thik we need to define a group.
But your design may constrain to define one. In that case, I would rather suggest "API monitoring".

Reply:
You're correct. Currently a group is needed for each and every channel (due to the way Channelhelpers works and current architecture is not meant to mix file base thing definition with dynamic created).